### PR TITLE
Contributor docs

### PR DESCRIPTION
**Pull request by wlach**
This is a codification of what is currently an informal policy. We should discuss this (at minimum) in the data team meeting before merging this PR.

/cc @chutten @kparlante

### CODE REVIEW

File: src/concepts/gaining_access.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Access to Firefox's telemetry data is limited to two groups:
+
+1. Mozilla employees and contractors
+2. Contributors who have signed an [NDA](https://wiki.mozilla.org/NDA),
```
Comment (mreid-moz) [inline src/concepts/gaining_access.md]:
```suggestion
2. Contributors who have signed a [non-disclosure agreement](https://wiki.mozilla.org/NDA),
```

Comment (mreid-moz) [reply src/concepts/gaining_access.md]: This would allow removing `NDA` from the spelling list

Comment (wlach) [reply src/concepts/gaining_access.md]: Great to remove jargon where we can! Thanks for the suggestion

File: .spelling [hunk 2]
```diff
@@ -149,6 +149,7 @@ natively
 navigations
 nbconvert
 nbviewer
+NDA
```
Comment (wlach) [inline .spelling]:
```suggestion
```

**Comment by wlach** [comment]
Tracking feedback here: https://bugzilla.mozilla.org/show_bug.cgi?id=1592117

**Pull request merged**